distributed/worker.py

```diff
@@ -2584,6 +2584,12 @@ def get_client(address=None, timeout=3):
 
     This client connects to the same scheduler to which the worker is connected
 
+    Submitting new jobs with the client returned by this function will likely
+    work best with ``client.compute``, especially if no computation can proceed
+    until task completion and the tasks take a significant amount of time.
+    However, calls to ``secede`` and ``rejoin`` `may` be more adventageous for
+    particular use cases.
```

Comment:
Why is client.compute better than client.submit or client.map or client.persist here?  Two issues:

1.  I don't think that this is true, (I think that you're thinking of dask.compute) 
2.  This is likely very situation-specific.  People use `get_client` in many different situations.  I've found that it's hard to give generic advice like this and have it not confuse people in the future.

Reply:
> I think that you're thinking of dask.compute

Thanks for catching that, I typo'd client.compute.

> This is likely very situation-specific ... it's hard to give generic advice

Maybe the docs should list common uses you've seen? i.e., something like "Submitting jobs with this client has seen a couple common uses: (1) using dask.compute, which does ... and results in .... This is most applicable when X or Y." Then maybe a section on warnings or links to SO questions?

This might be better as a separate doc page.

Reply:
Yes, moving this to a separate doc page might be nice.